### ISSUE TITLE
Support docker push command line options like --all-tags

### ISSUE THREAD

**Issue by jasonouellet-qc** [jfrog/artifactory-vsts-extension#469]
It's very frustrating to not be able to use tool usability in a tool wrapper like this.
The "docker push" and "docker pull" commands are universal and widely used options.

It doesn't make sense for me to call the tool 4 times in my pipeline to post 4 tags to the same generated image. This makes pipeline execution more cumbersome and inefficient.

My thinking started with the simple need to use the "--all-tags" flag of the [docker push](https://docs.docker.com/engine/reference/commandline/push/) command but I believe that the tools should be more open to using the other options of Docker's Pull and Push commands.

I propose adding an input variable in the "advanced" visual and adding the content of this variable to the command line for Pull and Push commands. Validation of the content will be at the user's risk because this type of option is dependent on the validation of the called command and this is not the purpose of this tool.

**Comment by cb645j** [comment]
i second this

**Comment by sgoschin** [comment]
yes agreed. I had the same issue was hoping to provide the flag....
